### PR TITLE
Player list added

### PR DESCRIPTION
**Pull request by Moondarker**

### CODE REVIEW

**Comment by Karang** [comment]
Run `npm run fix` to make the tests happy

**Comment by Moondarker** [comment]
Fixed it by hand, then saw your comment, good to know 👍

**Review by Karang — APPROVED** [review]

File: lib/playerlist.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        playerList.style.display = 'none'
+      }
+
+      self.isOpen = shouldShow
+    }
+
+    function addPlayer (player) {
+      const playerEntry = document.createElement('div') // There is likely a better way to handle this CSS-wise
+
+      const playerPingContainer = document.createElement('div')
```
Comment (rom1504) [inline lib/playerlist.js:100]: it's ok but just fyi, in lit element we have a templating language that is fast, so you can just do a loop in render function to achieve the same thing with nicer code

Comment (Moondarker) [reply lib/playerlist.js:100]: I might look into lit element's features and make it nicer later bc the way it looks rn really bothers me :)
I just blindly followed the way you adapted other parts this time tbh

Comment (rom1504) [reply lib/playerlist.js:100]: I didn't adapt other parts very well, just stuff things in components so it works, but it's not the idiomatic way
https://github.com/PrismarineJS/prismarine-web-client/blob/master/lib/loading_screen.js is a bit better

lit element has a nice tutorial https://lit-element.polymer-project.org/guide/templates

**Pull request merged**